scripts/variantstore/wdl/GvsVQSRClassic.wdl

```diff
@@ -373,21 +372,20 @@ task GatherTranches {
 
     count=0
     until cat $tranches_fofn | gsutil -m cp -L cp.log -c -I tranches/; do
-    sleep 1
-    ((count++)) && ((count >= $RETRY_LIMIT)) && break
+      sleep 1
+      ((count++)) && ((count >= $RETRY_LIMIT)) && break
     done
     if [ "$count" -ge "$RETRY_LIMIT" ]; then
-    echo 'Could not copy all the tranches from the cloud' && exit 1
+      echo 'Could not copy all the tranches from the cloud' && exit 1
     fi
 
     cat $tranches_fofn | rev | cut -d '/' -f 1 | rev | awk '{print "tranches/" $1}' > inputs.list
 
-    gatk --java-options -Xms6g \
-    GatherTranches \
-    --input inputs.list \
-    --mode ~{mode} \
-    -tranche ~{sep=' -tranche ' output_tranche_values} \
-    --output ~{output_filename}
+    gatk --java-options -Xms6g GatherTranches \
+      --input inputs.list \
+      --mode ~{mode} \
+      -tranche ~{sep=' -tranche ' output_tranche_values} \
```

Comment:
why dose this only have one dash and there are two of them? Probably out of scope but it still looks odd to me